kodilla-hibernate/src/main/java/com/kodilla/hibernate/manytomany/dao/facade/CompanyFacade.java

```diff
@@ -29,40 +29,32 @@
     CompanyFacade() {
     }
 
-    public ResultDto search(String parm) throws CompanyFacadeExpection {
-        if (StringUtils.isBlank(parm)) {
-            LOGGER.error(CompanyFacadeExpection.ERR_NULL_SEARCH);
-            throw new CompanyFacadeExpection(CompanyFacadeExpection.ERR_NULL_SEARCH);
+    public ResultDto search(String searched) throws CompanyFacadeException {
+        if (StringUtils.isBlank(searched)) {
+            LOGGER.error(CompanyFacadeException.ERR_NULL_SEARCH);
+            throw new CompanyFacadeException(CompanyFacadeException.ERR_NULL_SEARCH);
         }
 
-        List<Employee> employeeResultList = employeeDao.retrieveEmployeeByParmSearch(parm);
-        List<Company> companyResultList = companyDao.retrieveCompanyByParmSearch(parm);
+        List<Employee> employeeResultList = employeeDao.retrieveEmployeeByArgSearch(searched);
+        List<Company> companyResultList = companyDao.retrieveCompanyByArgSearch(searched);
 
-        try {
-            if (employeeResultList.size() == 0) {
-                LOGGER.info("No Employee find with this parameter");
-            } else {
-                for (Employee employee : employeeResultList) {
-                    LOGGER.info("Employee: " + employee.getLastname());
-                }
-            }
-        }finally {
-            resultDto.setEmployeesResult(employeeResultList);
+        if (employeeResultList.size() == 0) {
+            LOGGER.info("No Employee find with this parameter");
+        } else {
+            for (Employee employee : employeeResultList) {
+                LOGGER.info("Employee: " + employee.getLastname()); }
```

Comment:
czy to jest do czegokolwiek potrzebne? wyobraz sobie, jak beda wygladaly logi takiej aplikacji...